### ISSUE TITLE
flush_at_shutdown documentation

### ISSUE THREAD

**Issue by chicofranchico** [fluent/fluentd-docs#280]
http://docs.fluentd.org/v0.14/articles/buffer-section says it's default true for File but actually here http://docs.fluentd.org/v0.12/articles/out_file#buffertype it's default true for Memory

did that change between versions? thanks!

**Comment by fujimotos** [comment]
Sorry for late response.

> did that change between versions? thanks!

No, it didn't. AFAICT, the v0.14/v1.0 doc is just plain wrong.

In fact, I can confirm the `flush_at_shutdown` parameter gets turned off for file buffers by default in  v1.0. Here is the related lines in plugin/output.rb:

```ruby
def configure(conf)
  ...
  @flush_at_shutdown = @buffer_config.flush_at_shutdown
  if @flush_at_shutdown.nil?
    @flush_at_shutdown = if @buffer.persistent?
               false
             else
               true # flush_at_shutdown is true in default for on-memory buffer
             end
```

As you can see, the output plugin uses different default values depending on whether the buffer is persistent or not. Since FileBuffer is regarded as a persistent buffer (see plugin/buf_file.rb), `flush_at_shutdown` becomes false for this case.

**Comment by fujimotos** [comment]
And the v1.0 manual now gets fixed by the pull request #400.
So I'll close this issues...
